docs/production.md

```diff
@@ -73,8 +78,8 @@ commands will work with no further configuration.
 system, exposes the same API as a single Docker host, which means you can use
 Compose against a Swarm instance and run your apps across multiple hosts.
 
-Compose/Swarm integration is still in the experimental stage, but if you'd like
-to explore and experiment, check out the [integration guide](swarm.md).
+Read more about the Compose/Swarm integration on the
```

Comment:
Minor nitpick: change Read more about the Compose/Swarm **in** the Integration Guide (instead of "on"). Other than that, LGTM!